spectrum.js

```diff
@@ -244,6 +252,10 @@
             if (opts.showPaletteOnly) {
                 opts.showPalette = true;
             }
+            
+            if (opts.alphaVertical) {
+                colorControls.addClass("sp-vertical-alpha");
```

Comment:
This class should be added on the container element so we can select it for the sp-top thing as above